openeo/extra/job_management.py

```diff
@@ -189,14 +199,15 @@ def _normalize_df(self, df: pd.DataFrame) -> pd.DataFrame:
         required_with_default = [
             ("status", "not_started"),
             ("id", None),
-            ("start_time", None),
+            ("start_time", None), 
+            ("running_start_time", None), 
             # TODO: columns "cpu", "memory", "duration" are not referenced directly
             #       within MultiBackendJobManager making it confusing to claim they are required.
-            #       However, they are through assumptions about job "usage" metadata in `_update_statuses`.
+            #       However, they are through assumptions about job "usage" metadata in `_track_statuses`.
             ("cpu", None),
             ("memory", None),
             ("duration", None),
-            ("backend_name", None),
+            ("backend_name", None)
```

Comment:
```suggestion
            ("backend_name", None),
```
I would not remove this trailing comma (e.g. code formatters like [black](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#trailing-commas) will put it back anyway)